src/FormCollection/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```

Comment:
We *should* have this file, but it should match the contents from the other repos :) https://github.com/symfony/ux/blob/main/src/Dropzone/.gitattributes

Reply:
Yes, it's fixed